### ISSUE TITLE
itemjoin world at console produces an exception

### ISSUE THREAD

**Issue by SlimeDog** [RockinChaos/ItemJoin#356]
## Thanks for taking the time to open a bug report!

**Describe the bug**
`itemjoin world` at the console produces an exception

**To Reproduce**
1. Connect to console
2. itemjoin world

**Expected behavior**
A message that command not supported at the console, or is available only in-game.

```
>itemjoin world
[15:39:50] [Server thread/INFO]: ]------------------[ ItemJoin ]-----------------[
[15:39:50] [Server thread/INFO]: 
[15:39:50] [Server thread/WARN]: Unexpected exception while parsing console command "itemjoin world"
org.bukkit.command.CommandException: Unhandled exception executing command 'itemjoin' in plugin ItemJoin v5.1.1-SNAPSHOT-b603
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-1.16.5.jar-2021-01-21-2217:2993-Spigot-5bde311-b5eb2f5]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-1.16.5.jar-2021-01-21-2217:2993-Spigot-5bde311-b5eb2f5]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchCommand(CraftServer.java:763) ~[spigot-1.16.5.jar-2021-01-21-2217:2993-Spigot-5bde311-b5eb2f5]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchServerCommand(CraftServer.java:748) ~[spigot-1.16.5.jar-2021-01-21-2217:2993-Spigot-5bde311-b5eb2f5]
        at net.minecraft.server.v1_16_R3.DedicatedServer.handleCommandQueue(DedicatedServer.java:387) ~[spigot-1.16.5.jar-2021-01-21-2217:2993-Spigot-5bde311-b5eb2f5]
        at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:356) ~[spigot-1.16.5.jar-2021-01-21-2217:2993-Spigot-5bde311-b5eb2f5]
        at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1008) ~[spigot-1.16.5.jar-2021-01-21-2217:2993-Spigot-5bde311-b5eb2f5]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:847) ~[spigot-1.16.5.jar-2021-01-21-2217:2993-Spigot-5bde311-b5eb2f5]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:164) ~[spigot-1.16.5.jar-2021-01-21-2217:2993-Spigot-5bde311-b5eb2f5]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.ClassCastException: class org.bukkit.craftbukkit.v1_16_R3.command.ColouredConsoleSender cannot be cast to class org.bukkit.entity.Player (org.bukkit.craftbukkit.v1_16_R3.command.ColouredConsoleSender and org.bukkit.entity.Player are in unnamed module of loader 'app')
        at me.RockinChaos.itemjoin.ChatExecutor.onCommand(ChatExecutor.java:181) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-1.16.5.jar-2021-01-21-2217:2993-Spigot-5bde311-b5eb2f5]
        ... 9 more
```

**Please complete the following information:**
 - Resource Version and Build Number: 5.1.1-SNAPSHOT-b603
 - Server Version: Spigot-1.16.5

**Comment by RockinChaos** [comment]
Replicated, should now be fixed in the latest snapshot.

Link; https://ci.craftationgaming.com/job/ItemJoin/660/
**Remember to remove your old ItemJoin.jar file.**

**Comment by SlimeDog** [comment]
Thanks. Confirmed fixed in 5.1.1-SNAPSHOT-b660.
